src/server/req_manager.c

```diff
@@ -1738,9 +1741,12 @@ mgr_sched_unset(struct batch_request *preq)
 				t_list->al_link.ll_next->ll_struct = NULL;
 				rc = mgr_unset_attr(server.sv_attr, svr_attr_def, SRV_ATR_LAST, t_list,
 					-1, &bad_attr, (void *)&server, PARENT_TYPE_SERVER, INDIRECT_RES_CHECK);
-				if (rc != 0)
+				if (rc != 0) {
+					free_svrattrl(t_list);
 					reply_badattr(rc, bad_attr, tmp_plist, preq);
+				}
 				svr_save_db(&server, SVR_SAVE_FULL);
+				free_svrattrl(t_list);
```

Comment:
In the case of rc != 0, you're freeing t_list twice.

Reply:
Done